src/styles.css

```
@@ -754,3 +755,16 @@ span.metrics-counter {
   font-weight: 700;
   line-height: 1.2em;
 }
+
+.dropdown-menu {
+  --bs-dropdown-link-active-bg: #34495E;
+  overflow: hidden;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)